prep-javascript-variables/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>JavaScript Variables</title>
+</head>
+
+<body>
+
+  /* your code here */
+  <script>
+    var fullName = 'Joshua Lee Tenny';
```

Comment:
You want to declare all the variables in this exercise with `const` except for the never variable, that needs to be declared with `let` because it is not assigned a value when it is declared.